### ISSUE TITLE
The time stamp generated by command idevicesyslog is till the seconds value. Is it possible to get the timestamp till milliseconds value?

### ISSUE THREAD

**Issue by kk2212** [libimobiledevice/libimobiledevice#976]

**Comment by yyx112358** [comment]
> _No description provided._

Strongly approved. Millisecond timestamp is very useful to profile app performance.

**Comment by fbaptista** [comment]
Ohh come on!!!!! 🙏🥺

**Comment by nikias** [comment]
Yes, this works with a different service called `com.apple.os_trace_relay` (which is used by Apple's Console.app btw). I am working on support for this.

**Comment by warpfactorfive** [comment]
Ha!  Just researching this issue and found the issue/request.  Definitely a desirable enhancement.

**Comment by Dantee296** [comment]
@nikias do we have something on `com.apple.os_trace_relay` ?
